AirportTicketBookingSystem.Tests/DomainTests/ServicesTests/FlightServiceTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Moq;
+using FluentAssertions;
+using AutoFixture;
+using AirportTicketBookingSystem.Domain.Models;
+using AirportTicketBookingSystem.Domain.Repositories;
+using AirportTicketBookingSystem.Domain.Services;
+public class FlightServiceTests
+{
+    private readonly Mock<IFlightRepository> _mockFlightRepo;
+    private readonly FlightService _flightService;
+    private readonly Fixture _fixture;
+
+    public FlightServiceTests()
+    {
+        _mockFlightRepo = new Mock<IFlightRepository>();
+        _flightService = new FlightService(_mockFlightRepo.Object);
+        _fixture = new Fixture();
+    }
+
+    [Fact]
+    public void AddFlight_ShouldCallRepositoryAddFlight()
+    {
+        var flight = _fixture.Create<Flight>();
+
+        _flightService.AddFlight(flight);
+
+        _mockFlightRepo.Verify(repo => repo.AddFlight(flight), Times.Once);
+    }
+
+    [Fact]
+    public void AddFlight_ShouldThrowException_WhenFlightAlreadyExists()
+    {
+        var flight = _fixture.Create<Flight>();
+        _mockFlightRepo.Setup(repo => repo.AddFlight(It.IsAny<Flight>()))
+            .Throws(new InvalidOperationException($"\nBooking {flight.Id} already exists"));
+
+        Action act = () => _flightService.AddFlight(flight);
+
+        act.Should().Throw<InvalidOperationException>()
+            .WithMessage($"\nBooking {flight.Id} already exists");
+    }
+
+    [Fact]
+    public void GetAllFlights_ShouldReturnAllFlights()
```

Comment:
`GetAllFlights_ShouldReturnAllFlights_ByDefault`, for consistency `MethodName_StateUnderTest_ExpectedBehavior`. Same comment for the other tests.